src/components/common/Map.js

```diff
@@ -0,0 +1,129 @@
+import React, { PropTypes } from 'react';
+import { GoogleMapLoader, GoogleMap, Marker } from 'react-google-maps';
+import { getMyPosition } from '../../utils/MapHelper';
+
+class Map extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+
+    this.onMapClick = this.onMapClick.bind(this);
+    this.success = this.success.bind(this);
```

Comment:
I'd use more descriptive names. `getPositionSuccess`, `getPositionError`, etc. No need to change if you think it's okay as it is.